keeper/src/main/scala/zio/keeper/membership/swim/SWIM.scala

```diff
@@ -0,0 +1,85 @@
+package zio.keeper.membership.swim
+
+import izumi.reflect.Tags.Tag
+import zio._
+import zio.clock.Clock
+import zio.duration._
+import zio.keeper._
+import zio.keeper.discovery.Discovery
+import zio.keeper.membership.swim.protocols._
+import zio.keeper.membership.{ Membership, MembershipEvent, NodeAddress, TaggedCodec }
+import zio.logging.Logging.Logging
+import zio.logging._
+import zio.stream.ZStream
+
+object SWIM {
+
+  def run[B: TaggedCodec: Tag](
+    port: Int
+  ): ZLayer[Discovery with Logging with Clock, Error, Membership[B]] =
+    ZLayer.fromManaged(for {
+      _            <- log.info("starting SWIM on port: " + port).toManaged_
+      udpTransport <- transport.udp.live(64000).build.map(_.get)
```

Comment:
Perhaps pull out the configuration parameters (ttl, queue size, repeat timeout).

Reply:
as I mentioned I need to create config and extract those bits but don't want to do it as part of this PR because it's huge already